src/vueRecaptcha.vue

```diff
@@ -36,6 +41,7 @@
             else
                 return false;
         },
+        error: (reason: any) => reason,
```

Comment:
reason should be string.

Reply:
It could also be an `Error` since we emit from a `catch` block. We could coerce to a string for consistency if you prefer?